build/protobuf.mk

```diff
@@ -62,12 +62,6 @@ CPP_PROTOS := $(filter %/roachpb/metadata.proto %/roachpb/data.proto %/roachpb/i
 CPP_HEADERS := $(subst ./,$(NATIVE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.h))
 CPP_SOURCES := $(subst ./,$(NATIVE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.cc))
 
-$(PROTOC_PLUGIN):
-	$(GO_INSTALL) $(REPO_ROOT)/pkg/cmd/protoc-gen-gogoroach
-
-$(GRPC_GATEWAY_PLUGIN):
```

Comment:
It appears this plugin is still used. Is there another build rule taking care of this now? (e.g. this got replaced by a `pkg/cmd/...` command)

Reply:
Yeah, it's included in the $(BOOTSTRAP_TARGET) recipe, so including it here was redundant. You should definitely check my eyes on this, though.

Reply:
👍 looks good.